go/vt/sqlannotation/sqlannotation.go

```diff
@@ -67,51 +68,54 @@ func IsDML(sql string) bool {
 		strings.EqualFold(word, "delete")
 }
 
-// ExtractKeySpaceID parses the annotation of the given statement and tries
+// ExtractKeySpaceIDS parses the annotation of the given statement and tries
 // to extract the keyspace id.
 // If a keyspace-id comment exists 'keyspaceID' is set to the parsed keyspace id
 // and err is set to nil; otherwise, if a filtered-replication-unfriendly comment exists
 // or some other parsing error occured, keyspaceID is set to nil and err is set to a non-nil
 // error value.
-func ExtractKeySpaceID(sql string) (keyspaceID []byte, err error) {
-	keyspaceIDString, hasKeySpaceID := extractStringBetween(sql, "/* vtgate:: keyspace_id:", " ")
+func ExtractKeySpaceIDS(sql string) (keyspaceIDs [][]byte, err error) {
```

Comment:
We use Keyspace as a word, not as two words, so it should be 'Keyspace' not 'KeySpace', in this function name, and below in 'hasKeySpaceID'.